src/Catalyst.Core.Lib/P2P/ReputationSystem/ReputationManager.cs

```diff
@@ -41,7 +42,6 @@ public sealed class ReputationManager : IReputationManager, IDisposable
         public readonly ReplaySubject<IPeerReputationChange> ReputationEvent;
         public IObservable<IPeerReputationChange> ReputationEventStream => ReputationEvent.AsObservable();
         public IObservable<IPeerReputationChange> MergedEventStream { get; set; }
```

Comment:
Again a bit of backstory as to why we want to remove these? I think this was there to stop a previous threading issue, what is autofacs lifetime scope for injecting this to the application.